cmd/package.go

```diff
@@ -688,3 +687,13 @@ func constructBootCmdFromArguments(commandLine, customBoot, packageDir string) s
 	fmt.Println("Empty command line will be set for this image")
 	return ""
 }
+
+// AbsTarPathMatches tells whether the tar header name matches the path.
+// This function is needed since some tar files prefix its header names
+// with / and some not. NOTE: argument 'path' should start with '/' while
+// 'tarPath' should be header.Name.
+func AbsTarPathMatches(path, tarPath string) (res bool) {
```

Comment:
Furthermore, I had to check many times yesterday the order of params. I think `tarPath, pathPattern` would be more natural.